govtool/frontend/src/components/organisms/AutomatedVotingOptions.tsx

```diff
@@ -40,6 +41,25 @@ export const AutomatedVotingOptions = ({
   // TODO: Change to certain automated voted option if available
   const onClickInfo = () => openInNewTab("https://docs.sanchogov.tools/");
 
+  const isDelegatedToAbstain = currentDelegation === "drep_always_abstain";
```

Comment:
can this delegation keys be moved to const or enum somewhere? Comparing values with string is always a bit risky